src/metabase/models/dashboard.clj

```diff
@@ -18,17 +19,53 @@
   (db/cascade-delete! 'Revision :model "Dashboard" :model_id id)
   (db/cascade-delete! DashboardCard :dashboard_id id))
 
+(defn- pre-insert [dashboard]
+  (let [defaults {:parameters []}]
+    (merge defaults dashboard)))
```

Comment:
If `[]` is the column-level default value of `:parameters` we shouldn't need this step. It would be a little faster to let the DB take care of it